### DISCUSSION TITLE
Membership Application

### DISCUSSION THREAD

**Discussion by matejnesuta** [kiali/kiali#5665] [Governance]
Hello
I have been a member of the OSSM Quality Engineering team for the past 6 months. My main focus is the Kiali Cypress E2E suite. I have helped with the migration and improvement of the original Selenium suite. I would like to become a tester, mainly to be able to request reviews and also to give reviews to other tests.

**Comment by ScriptingShrimp** [comment]
Matej's contributions are significant. He is contributing to tests over 4 months now. 
Take a look [here!](https://github.com/kiali/kiali/pulls?q=is%3Apr+author%3Amatejnesuta+)
Please share your opinions @kiali/maintainers
